File: client/trunk/src_lua/app/conf/arena_shopConfig.lua
-- 自动生成，请勿修改 
-- 时间：2016/05/18
-- 21102585@qq.com

local arena_shopConfig = class("arena_shopConfig")
function arena_shopConfig:ctor()
	self.fields = {"key", "limit_count", "goods_id", "reputation"}
	self.datas = {
		[1] = {1, 10, 110049, 10},
		[2] = {2, 100, 110078, 2},
		[3] = {3, 10, 110055, 20},
		[4] = {4, 10, 110079, 10},

	}
end
return arena_shopConfig

File: server/trunk/server/src/config/arena_shop_config.erl
%%%-------------------------------------------------------------------
%%% @author zhengsiying
%%% @doc
%%%		自动生成文件，不要手动修改
%%% @end
%%% Created : 2016/10/12
%%%-------------------------------------------------------------------

-module(arena_shop_config).

-include("common.hrl").
-include("config.hrl").

-compile([export_all]).

get_list_conf() ->
	[ arena_shop_config:get(X) || X <- get_list() ].

get_list() ->
	[1, 2, 3, 4].

get(1) ->
	#arena_shop_conf{
		key = 1,
		limit_count = 10,
		goods_id = 110049,
		reputation = 10
	};

get(2) ->
	#arena_shop_conf{
		key = 2,
		limit_count = 100,
		goods_id = 110078,
		reputation = 2
	};

get(3) ->
	#arena_shop_conf{
		key = 3,
		limit_count = 10,
		goods_id = 110055,
		reputation = 20
	};

get(4) ->
	#arena_shop_conf{
		key = 4,
		limit_count = 10,
		goods_id = 110079,
		reputation = 10
	};

get(_Key) ->
	?ERR("undefined key from arena_shop_config ~p", [_Key]).